frontend/ui/widget/inputdialog.lua

```diff
@@ -455,6 +453,21 @@ function InputDialog:init()
         },
         frame
     }
+    if Device:isTouchDevice() then -- is used to hide the keyboard with a tap outside of inputbox
+        local GestureRange = require("ui/gesturerange")
```

Comment:
You can move that among the require at top (I did put it there just to limit the diff size).